ethpm/utils/registry.py

```diff
@@ -1,43 +1,20 @@
-from urllib import parse
-
 from typing import List
 
 from ethpm.exceptions import UriNotSupportedError
 
-from ethpm.validation import (
-    validate_package_name,
-    validate_registry_uri_authority,
-    validate_registry_uri_scheme,
-    validate_registry_uri_version,
-)
-
 
-def is_ens_domain(auth: str) -> bool:
+def is_ens_domain(authority: str) -> bool:
```

Comment:
Not part of this PR but in general the docstrings should use imperative, present tens.

Helper: When the `is_ens_domain` is called it will:

>Return `False` if URI authority is not a valid ENS domain.